### PR TITLE
feature/swodlr-ui-56: add BrowserRouter

### PR DESCRIPTION
**Pull request by jbyrne6**
### Changes
- Changed from HashRouter to BrowserRouter

### CODE REVIEW

File: src/components/app/App.tsx [hunk 1]
```diff
@@ -23,7 +23,9 @@ const App = () => {
     // check if user is authenticated
     useEffect(() => {
       // navigate to login page if user isn't authenticated
-      navigate('/')
+      // if (!userAuthenticated) {
```
Comment (frankinspace) [inline src/components/app/App.tsx:26]: Remove commented out code

File: src/components/sidebar/ProductCustomization.tsx [hunk 2]
```diff
@@ -18,26 +18,72 @@ const ProductCustomization = () => {
...
-            } else {
-                newParamsObject['rasterResolutionUTM'] = parameterOptionValues.rasterResolutionUTM.default
-            }
-        }
-        if (!search.includes('showUTMAdvancedOptions')) {
-            newParamsObject['showUTMAdvancedOptions'] = false
-        }
-        addSearchParamToCurrentUrlState(newParamsObject)
-        const parametersFromUrl = {...generateProductParameters, ...newParamsObject}
+        // const newParamsObject: newUrlParamsObject = {}
```
Comment (frankinspace) [inline src/components/sidebar/ProductCustomization.tsx:58]: Remove commented out code

**Review by frankinspace — CHANGES_REQUESTED** [review]
When adding granules you get a success message along with multiple error messages. See short clip:

https://github.com/podaac/swodlr-ui/assets/89428916/641ed57b-8d0c-45c5-aff0-9bc581ff6b7e

File: src/components/sidebar/ProductCustomization.tsx (hunk 2 above)
Comment (jbyrne6) [reply src/components/sidebar/ProductCustomization.tsx]: done

File: src/components/app/App.tsx (hunk 1 above)
Comment (jbyrne6) [reply src/components/app/App.tsx:26]: done

**Comment by jbyrne6** [comment]
> When adding granules you get a success message along with multiple error messages. See short clip:
> 
>  adding-granules.mov

Found that the useEffect in the GranuleTable.tsx page was re-rendering on change of the addedProducts redux state variable so removed that re-render condition to solve this incorrect alert issue.

**Comment by jbyrne6** [comment]
Bug: noticed that in SIT, the band and zone adjust is not matching on the site what it says in the url. Also look into bug with adding multiple scenes at once with a hyphen

**Review by frankinspace — APPROVED** [review]

**Pull request merged**